examples/framework-lit/src/pages/index.astro

```diff
@@ -0,0 +1,21 @@
+---
+import Lorem from '../components/Lorem.astro';
+import '../components/Test.js';
```

Comment:
Is there not a way to import similar to other components where you name the imported component? I guess this is the general way they're written... just seems weird to me 😅

Reply:
I guess you can, since you do that later...

Reply:
This is weird because `custom-elements` are defined as a side-effect, so we had to work around that limitation.

Regular `Component`-style usage is possible, but we heard some feedback that most folks would prefer to use `kebab-case` for their custom elements, so this allows us to support that.

Reply:
Ah, so using the kebab case requires importing them as a side effect :0 That's fun. I guess the docs should hopefully show how to use them both ways though, right?

Reply:
Going to merge, but happy to continue the conversation.